typedoc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+	name: "webpack-cli",
+	mode: "modules",
+	out: "docs",
+	theme: "default",
```

Comment:
if the theme is default, can't we discard the prop?

Reply:
Agreed. Cleaned.